aws-athena/views/default-vw_pin_sale.sql

```diff
@@ -185,7 +185,8 @@ mydec_sales AS (
             ) > 0 AS sale_filter_ptax_flag,
             COUNT() OVER (
                 PARTITION BY line_1_primary_pin, line_4_instrument_date
-            ) AS num_cards_sale
+            ) AS num_cards_sale,
+            year_of_sale
```

Comment:
This missing column was causing `dbt run` to fail to build this model:

```
18:53:31  Runtime Error in model vw_pin_sale_test (models/default/vw_pin_sale_test.sql)
18:53:31    line 212:18: Column 'mydec_sales.year_of_sale' cannot be resolved
```

I had forgotten that I had made this adjustment prior to factoring out shared logic between the Athena and dbt views in https://github.com/ccao-data/data-architecture/pull/41.

Perhaps this change should also be pulled into `master`? It seems like the Athena version of the view correctly selects this column, so it must just be a discrepancy between the production version of the view and the version stored in this file. (Another nice category of problem that dbt will eventually resolve for us.)

Reply:
Can you cherry-pick this commit and open a PR to master? This seems like a mistake

Reply:
Can do! [PR up here](https://github.com/ccao-data/data-architecture/pull/48), I'll request your review once tests pass.

Reply:
Merged to `master` 👍🏻